src/Microsoft.AspNet.Mvc.Core/ConsumesAttribute.cs

```diff
@@ -30,6 +32,15 @@ public ConsumesAttribute(string contentType, params string[] otherContentTypes)
                 throw new ArgumentNullException(nameof(contentType));
             }
 
+            // We want to ensure that the given provided content types are valid values, so
+            // we validate them using the semantics of MediaTypeHeaderValue.
+            MediaTypeHeaderValue.Parse(contentType);
+
+            for (var i = 0; i < otherContentTypes.Length; i++)
+            {
+                MediaTypeHeaderValue.Parse(otherContentTypes[i]);
+            }
```

Comment:
Previous code ignored invalid media types. Why is it now correct to throw `FormatException` in that case?